kolibri/core/assets/src/views/breadcrumbs/index.vue

```diff
@@ -1,43 +1,64 @@
 <template>
 
-  <nav class="breadcrumbs">
-
-    <div v-if="collapsedCrumbs.length" class="breadcrumbs-dropdown-wrapper">
-      <ui-icon-button :has-dropdown="true" icon="expand_more" size="small">
-        <div slot="dropdown" class="breadcrumbs-dropdown">
-          <ol class="breadcrumbs-dropdown-items">
-            <li v-for="crumb in collapsedCrumbs" class="breadcrumbs-dropdown-item">
-              <router-link :to="crumb.link" :style="{ maxWidth: `${collapsedCrumbMaxWidth}px` }">
-                {{ crumb.text }}
-              </router-link>
-            </li>
-          </ol>
-        </div>
-      </ui-icon-button>
+  <div>
+    <nav class="breadcrumbs">
+      <div v-if="collapsedCrumbs.length" class="breadcrumbs-dropdown-wrapper">
+        <ui-icon-button :has-dropdown="true" icon="expand_more" size="small">
+          <div slot="dropdown" class="breadcrumbs-dropdown">
+            <ol class="breadcrumbs-dropdown-items">
+              <li v-for="crumb in collapsedCrumbs" class="breadcrumbs-dropdown-item">
+                <router-link :to="crumb.link" :style="{ maxWidth: `${collapsedCrumbMaxWidth}px` }">
+                  {{ crumb.text }}
+                </router-link>
+              </li>
+            </ol>
+          </div>
+        </ui-icon-button>
+      </div>
+
+      <ol class="breadcrumbs-visible-items">
+        <template v-for="(crumb, index) in crumbs">
+          <li
+            v-if="index !== crumbs.length - 1"
+            class="breadcrumbs-visible-item breadcrumbs-visible-item-notlast"
+            v-show="!crumb.collapsed"
+          >
+            <router-link :to="crumb.link">{{ crumb.text }}</router-link>
+          </li>
+
+          <li
+            v-else
+            class="breadcrumbs-visible-item breadcrumb-visible-item-last"
+          >
+            <span :style="{ maxWidth: `${lastCrumbMaxWidth}px` }">{{ crumb.text }}</span>
+          </li>
+        </template>
+      </ol>
+    </nav>
+
+
+    <div class="breadcrumbs breadcrumbs-offscreen">
+      <ol class="breadcrumbs-visible-items">
+        <template v-for="(crumb, index) in crumbs">
+          <li
+            v-if="index !== crumbs.length - 1"
+            :ref="`crumb${index}`"
+            class="breadcrumbs-visible-item breadcrumbs-visible-item-notlast"
+          >
+            <router-link :to="crumb.link" tabindex="-1">{{ crumb.text }}</router-link>
```

Comment:
Added tabindex to avoid tabbing, Any other markup I should add to make it `inaccessible`. :sweat_smile: